deployment-k8s/values.yaml

```diff
@@ -36,7 +36,8 @@ global:
     performance-test:
     - name: "SPRING_PROFILES_ACTIVE"
       value: "k8s,example-perf-k8s,nologs"
-    s3:
+    example-s3:
+    # The order of these properties is important, they are addressed by index in the aws-s3-k8s deployServicesToK8s.sh script
```

Comment:
Would it be possible to use a map instead of an array?

Reply:
it would work, but would require a lot of updates to our charts, and the way we load environment variables, and therefore isnt worth it